doc/src/build-instructions.md

```diff
@@ -732,19 +732,20 @@ If building for a different target USART may be configure as explained above in
 
 After your application has been tested (_and debugged_) and is ready to put into active use you may want to tune the build of AtomVM.  For instance disabling logging with `-DAVM_LOG_DISABLE=on` as a `cmake` configuration option may result in slightly better performance. This will have no affect on the console output of your application, just disable low level log messages from the AtomVM system. You may also want to enabling automatic reboot in the case that your application ever exits with a return other than `ok`. This can be enabled with the `cmake` option `-DAVM_CONFIG_REBOOT_ON_NOT_OK=on`.
 
-## Building for Raspberry Pi Pico
+## Building for Raspberry Pi Pico and Pico 2
```

Comment:
Should we keep implicit the 2040 name, or should we use the board name?

Reply:
This part has been retitled "Building for Raspberry Pi RP2" following the approach described above.
The only place I kept Pico and Pico 2 in the documentation is in the getting started where we refer explicitely to the board names.